crates/std_detect/src/detect/cache.rs

```diff
@@ -121,31 +121,11 @@ impl Cache {
     }
 }
 
-cfg_if::cfg_if! {
```

Comment:
why do you want to remove it?

Reply:
- It's never used.
- It's never tested.
- In fact the feature flag is even missing from `Cargo.toml`.

Reply:
I changed `std_detect_env_override` to use libc instead of std.

Reply:
Thank you a lot!

Reply:
Does this work on Windows? I did guess not.